packages/main/test/specs/Carousel.spec.js

```diff
@@ -0,0 +1,49 @@
+const assert = require("chai").assert;
+
+
+describe("Carousel general interaction", () => {
+	browser.url("http://localhost:8080/test-resources/pages/Carousel.html");
+
+	it("Carousel is rendered", () => {
+		const carouselRoot = browser.$("#carousel1").shadow$(".ui5-carousel-root");
+
+		assert.ok(carouselRoot, "Carousel is rendered.");
+	});
+
+	it("Carousel navigates left", () => {
+		const carousel = browser.$("#carousel1");
+		const carouselRightButton = carousel.shadow$$(".ui5-carousel-navigation-button")[0];
+
+		carouselRightButton.click();
+		assert.equal(carousel.getAttribute("selected-index"), "3", "Second view is in place");
+	});
+
+	it("Carousel navigates right", () => {
+		const carousel = browser.$("#carousel1");
+		const carouselRightButton = carousel.shadow$$(".ui5-carousel-navigation-button")[1];
+
+		carouselRightButton.click();
+		assert.equal(carousel.getAttribute("selected-index"), "1", "Second view is in place");
+	});
+
+	it("Navigation is rendered for carousel with less than 9 elements", () => {
+		const carousel = browser.$("#carousel1");
+		const navigation = carousel.shadow$(".ui5-carousel-navigation > div");
+
+		assert.ok(navigation, "Navigation is rendered");
```

Comment:
I remember that the way to check if DOM element exists is using the isExisting() method.
assert.ok(navigation.isExisting()) (same for the next test)